sklearn/model_selection/tests/test_validation.py

```diff
@@ -1210,9 +1207,13 @@ def test_learning_curve_remove_duplicate_sample_sizes():
                                n_redundant=0, n_classes=2,
                                n_clusters_per_class=1, random_state=0)
     estimator = MockImprovingEstimator(2)
-    train_sizes, _, _ = assert_warns(
-        RuntimeWarning, learning_curve, estimator, X, y, cv=3,
-        train_sizes=np.linspace(0.33, 1.0, 3))
+    warning_message = (
+        "Removed duplicate entries from 'train_sizes'. Number of ticks "
+        "will be less than the size of 'train_sizes' 2 instead of 3\\)."
```

Comment:
I don't understand why we have an imbalanced parenthesis in this message. We could fix the the original error message as part of this PR to remove the closing parenthesis.

Reply:
Also while we are at it we could insert `: `  before the end of the message: "2 instead of 3".

Reply:
I fixed the original message as well as the test one.